packages/postcss/mixin.core.js

```diff
@@ -9,9 +9,9 @@ const { join, trimSlashes } = require('pathifist');
 const getAssetPath = (...args) => trimSlashes(join(...args));
 
 const cssLoaderLocalOptions = {
-  camelCase: true,
-  modules: true,
-  localIdentName: '[folder]-[name]-[local]-[hash:8]',
+  modules: {
+    localIdentName: '[folder]-[name]-[local]-[hash:8]',
+  },
```

Comment:
I guess we have to add `localsConvention: 'camelCase'` here, since the default value is `asIs`, which will not convert e.g. kebab-case classnames to camel-case anymore.